book-crawler/scripts/create_tables.py

```
@@ -0,0 +1,59 @@
+def check_and_create_tables():
+    """테이블 확인 및 생성"""
+    
+    # 인스펙터를 사용하여 기존 테이블 확인
+    inspector = inspect(engine)
+    existing_tables = inspector.get_table_names()
+    
+    print("=== 데이터베이스 테이블 관리 ===")
+    print(f"현재 데이터베이스: {engine.url.database}")
+    
+    if existing_tables:
+        print(f"\n기존 테이블 목록:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)